package.xml

```diff
@@ -24,6 +24,7 @@
   <depend>nav_msgs</depend>
   <depend>std_srvs</depend>
   <depend>atos_interfaces</depend>
+  <depend>gui</depend>
```

Comment:
can it be called maybe atos_gui or sth? gui is a very generic package name

Reply:
Agree, I renamed it.